examples/simple_logging/simple_logging.ino

```diff
@@ -281,6 +281,22 @@ const char *DOptoDI12address = "5";  // The SDI-12 Address of the Zebra Tech D-O
 // const int8_t SDI12Power = 22;  // Pin to switch power on and off (-1 if unconnected)
 ZebraTechDOpto dopto(*DOptoDI12address, SDI12Power, SDI12Data);
 
+#include <ExternalVoltage.h>
+const int8_t VoltPower = 22;  // Pin to switch power on and off (-1 if unconnected)
+const int8_t VoltData = 0;  // The data pin ON THE ADS1115 (NOT the Arduino Pin Number)
+const uint8_t Volt_ADS1115Address = 0x48;  // The I2C address of the ADS1115 ADC
+ExternalVoltage extvolt(VoltPower, VoltData);
+
+#include <TippingBucket.h>
```

Comment:
@bschulz1701, this example code looks like it needs updating to include the I2C address.

Reply:
Fixed as of https://github.com/EnviroDIY/ModularSensors/pull/114/commits/872fe46629247bfde5cca95d6726f4c6cb7022a5